psi-ms.obo

```diff
@@ -22655,6 +22655,386 @@ relationship: has_metric_category MS:4000012 ! single run based metric
 relationship: has_value_type xsd:int ! The allowed value-type for this CV term
 relationship: has_units UO:0000189 ! count unit
 
+[Term]
+id: MS:4000108
+name: Peak density distribution MS1 Q1, Q2, Q3
+def: "From the distribution of peak densities in MS1, the quartiles Q1, Q2, Q3" [PSI:MS]
+comment: The distribution of peak densities in MS1 can provide insight into the presence of nuisance factors.
+is_a: MS:4000004 ! n-tuple
+relationship: has_metric_category MS:4000009 ! ID free metric
+relationship: has_value_concept STATO:0000167 ! first quartile
+relationship: has_value_concept STATO:0000574 ! center value
+relationship: has_value_concept STATO:0000170 ! third quartile
+relationship: has_value_type xsd:float ! The allowed value-type for this CV term
+	
+[Term]
+id: MS:4000109
+name: Peak density distribution MS1 mean
+def: "From the distribution of peak densities in MS1, the mean" [PSI:MS]
+comment: The distribution of peak densities in MS1 can provide insight into the presence of nuisance factors.
+is_a: MS:4000003 ! single value
+relationship: has_metric_category MS:4000009 ! ID free metric
+relationship: has_value_concept STATO:0000401 ! sample mean
+relationship: has_value_type xsd:float ! The allowed value-type for this CV term
+	
+[Term]
+id: MS:4000110
+name: Peak density distribution MS1 sigma
+def: "From the distribution of peak densities in MS1, the sigma value" [PSI:MS]
+comment: The distribution of peak densities in MS1 can provide insight into the presence of nuisance factors.
+is_a: MS:4000003 ! single value
+relationship: has_metric_category MS:4000009 ! ID free metric
+relationship: has_value_concept STATO:0000237 ! standard deviation
+relationship: has_value_type xsd:float ! The allowed value-type for this CV term
+	
+[Term]
+id: MS:4000111
+name: Peak density distribution MS1 low outliers (<Q1-1.5*IQR)
```

Comment:
**suggestion:** I agree with @edeutsch that we probably don't need to specify how outliers are determined in the term name. Similarly, it might be sufficient to recommend using 1.5 IQR as the threshold for outliers, rather than require it. Ultimately, what is an outlier depends on the data distribution, and in some cases you might use different thresholds.

Reply:
I think in the box-whisker display 1.5IQR is pretty much the standard but I do *not* disagree with you, prob better to leave it out of the name. Leaving the threshold to the user might pose some serious issues downstream, let's also discuss in next telco

Reply:
For e.g. submissions to Nature journals you now need to specify in all figure captions what the whiskers of box plots indicate, so it seems like the field is moving toward being more explicit rather than assuming that it's the default definition of 1.5 IQR. I found it a bit weird/annoying at first as well.